src/Tasks/PageIterator.php

```diff
@@ -26,18 +28,20 @@ class PageIterator
     private ?array $requestOptions = [];
 
     /**
-     * @param $res
+     * @param mixed $response
```

Comment:
```suggestion
     * @param mixed $response paged collection response
```

Reply:
I think we might other cases that are not catered for by the union type.

Reply:
I think leaving it as `mixed` is too open to the point a developer isn't sure what exactly to pass.

Was the discussion on whether PageIterator should work with Parsable's only or both Parsable & array/object resolved?